python/web/index.py

```diff
@@ -0,0 +1,202 @@
+import os
+import flask
+import gevent.pywsgi
+import subprocess
+import sys
+from io import StringIO
+from io import TextIOWrapper, BytesIO
+
+import examples
+# from .list import create_list
+
+app = flask.Flask(__name__)
+
+
+@app.route('/python/')
+def root():
+    return create_list()
+
+
+def file_get_contents(example):
+    current_example = os.getcwd() + '/examples/{0}.py'.format(example)
+
+    text = open(current_example, "r")
+    contents = text.read()
+    text.close()
+    return contents
+
+
+def capture_output(example):
+
+    file = os.getcwd() + '/examples/' + example + '.py'
+
+    # # setup the environment
+    # old_stdout = sys.stdout
+    # sys.stdout = TextIOWrapper(BytesIO(), sys.stdout.encoding)
+    #
+    # # do some writing (indirectly)
+    # # write("blub")
+    # os.system('python ' + file)
+    #
+    # # get output
+    # sys.stdout.seek(0)  # jump to the start
+    # out = sys.stdout.read()  # read output
+    #
+    # # restore stdout
+    # sys.stdout.close()
+    # sys.stdout = old_stdout
```

Comment:
Vestigial?

Reply:
Yes. All of this was me playing around with trying to pull in and run the python file by itself without using imports, so that it would not be necessary to include the `test_output()` function call in the outputted Source. If you're cool with it being there, which I think is cleaner and more pythonic in the end, I'll kill all these lines.